webpack.config.js

```diff
@@ -0,0 +1,13 @@
+const path = require('path');
+
+module.exports = {
+  entry: './js/index.js',
+  output: {
+    path: path.resolve(__dirname, 'www'),
+    filename: 'index.js'
+  },
+  mode: 'development',
+  node: {
+    fs: 'empty'
+  }
```

Comment:
I had to add the same module replace plugin that is used for the ionic example to make these configurations work:

```js
const webpack = require('webpack');

plugins: [
    new webpack.NormalModuleReplacementPlugin(/typeorm$/, function (result) {
      result.request = result.request.replace(/typeorm/, "typeorm/browser");
    }),
 ]
```
Otherwise webpack will load the "normal" version instead of the browser version.

Reply:
Fixed.